code/modules/ordeals/green_ordeals.dm

```diff
@@ -41,8 +41,11 @@
 
 /datum/ordeal/green_dusk/Run()
 	..()
+	var/list/availablespawns = list(GLOB.xeno_spawn)
```

Comment:
```suggestion
	var/list/availablespawns = GLOB.xeno_spawn.Copy()
```
What you did, was essentially a list(list())

Reply:
I forgot that it should just be an = and not =list(Glob.spawn)